src/components/projects/projectId/attributes/attributeId/ExecutionContainer.tsx

```diff
@@ -75,7 +75,16 @@ export default function ExecutionContainer(props: ExecutionContainerProps) {
     const sampleRecordsFinal = useMemo(() => {
         if (sampleRecords && sampleRecords.calculatedAttributesDisplay) {
             return sampleRecords.calculatedAttributesDisplay.map((record: any, index) => {
-                const calculatedValue = currentAttributesRef.current.dataType != DataTypeEnum.EMBEDDING_LIST ? record : { id: record.id, value: JSON.parse(record.value) }
+                let calculatedValue;
+                if (currentAttributesRef.current.dataType == DataTypeEnum.EMBEDDING_LIST) {
+                    calculatedValue = { id: record.id, value: JSON.parse(record.value) };
+                }
+                if (currentAttributesRef.current.dataType == DataTypeEnum.TEXT_LIST) {
+                    calculatedValue = { id: record.id, value: JSON.stringify(record) };
```

Comment:
i think record.value but i might be mistaken

Reply:
i will make them the same way